### DISCUSSION TITLE
what layout to chose when my computer is by default colemak dh

### DISCUSSION THREAD

**Discussion by Mistorcha** [manna-harbour/miryoku#324] [Q&A] [answered]
Hi, for what I understood, keyboard with microcontrollers where supposed to send the correct key to computer directly, but in fact it send keys assuming the fact that the layout on the computer is qwerty.

Considering I'm using several keyboards depending on "reasons", I have my base layout to be colemak dh, meaning that if I connect my corne with myrioku, It doesn't send the keys I want.
Being a smartass, I tried to change the myrioku layout to qwerty, but it doesnt work either (the bottom row loses the Z and gives me xcdv\ instead of zxcdv), this is most probably due to my colemak dh layout using ISO format leftmost key on the bottom row and qwerty not using this leftmost key. 

Does anyone know what layout I should chose for my myrioku built for it to work nicely with my computer in colemak dh?

Many thanks in advance for your help and time.

**Comment by manna-harbour** [comment, marked as answer]
It's similar to the situation when using localised layouts on the host. See https://github.com/manna-harbour/miryoku/discussions/197. In this case you'd need to start with QWERTY and substitute custom layers with https://github.com/manna-harbour/miryoku/discussions/85 to account for the differences, e.g. https://github.com/manna-harbour/miryoku/discussions/182, https://github.com/manna-harbour/miryoku/discussions/153. However, it's generally recommended to leave the host as QWERTY and do the mapping on the keyboard.
